v1/service_uuids.json

```diff
@@ -0,0 +1,78 @@
+    { "name": "Insulin Delivery", "identifier": "org.bluetooth.service.insulin_delivery" , "uuid": "183A" , "source": "gss"},
+    { "name": "Internet Protocol Support Service", "identifier": "org.bluetooth.service.internet_protocol_support" , "uuid": "1820" , "source": "gss"},
+    { "name": "Link Loss", "identifier": "org.bluetooth.service.link_loss" , "uuid": "1803" , "source": "gss"},
+    { "name": "Location and Navigation","identifier": "org.bluetooth.service.location_and_navigation" , "uuid": "1819" , "source": "gss"},
+    { "name": "Mesh Provisioning Service", "identifier": "org.bluetooth.service.mesh_provisioning" , "uuid": "1827" , "source": "gss"},
+    { "name": "Mesh Proxy Service", "identifier": " org.bluetooth.service.mesh_proxy" , "uuid": "1828" , "source": "gss"},
+    { "name": "Next DST Change Service", "identifier": "org.bluetooth.service.next_dst_change" , "uuid": "1807" , "source": "gss"},
+    { "name": "Object Transfer Service", "identifier": "org.bluetooth.service.object_transfer" , "uuid": "1825" , "source": "gss"},
+    { "name": "Phone Alert Status Service", "identifier": "org.bluetooth.service.phone_alert_status" , "uuid": "180E" , "source": "gss"},
+    { "name": "Pulse Oximeter Service", "identifier": "org.bluetooth.service.pulse_oximeter" , "uuid": "1822" , "source": "gss"},
+    { "name": "Reconnection Configuration", "identifier": "org.bluetooth.service.reconnection_configuration" , "uuid": "1829" , "source": "gss"},
+    { "name": "Reference Time Update Service", "identifier": "org.bluetooth.service.reference_time_update" , "uuid": "1806" , "source": "gss"},
+    { "name": "Running Speed and Cadence", "identifier": "org.bluetooth.service.running_speed_and_cadence" , "uuid": "1814" , "source": "gss"},
+    { "name": "Scan Parameters", "identifier": "org.bluetooth.service.scan_parameters" , "uuid": "1813" , "source": "gss"},
+    { "name": "Transport Discovery", "identifier": "org.bluetooth.service.transport_discovery" , "uuid": "1824" , "source": "gss"},
+    { "name": "Tx Power", "identifier": "org.bluetooth.service.tx_power" , "uuid": "1804" , "source": "gss"},
+    { "name": "User Data", "identifier": "org.bluetooth.service.user_data" , "uuid": "181C" , "source": "gss"},
+    { "name": "Weight Scale", "identifier": "org.bluetooth.service.weight_scale", "uuid": "181D" , "source": "gss"},
+
+    { "name": "Apple Notification Center Service", "identifier": "com.apple.service.notification_center", "uuid": "7905F431-B5CE-4E99-A40F-4B1E122D00D0" , "source": "apple"},
+    { "name": "Apple Media Service", "identifier": "com.apple.service.media", "uuid": "89D3502B-0F36-433A-8EF4-C502AD55F8DC" , "source": "apple"},
+
+    { "name": "micro:bit Accelerometer Service", "identifier": "com.micro.bit.service.accelerometer", "uuid": "E95D0753-251D-470A-A062-FA1922DFA9A8" , "source": "micro:bit"},
```

Comment:
Maybe not important, but could be good to align identifier with official domain names. For microbit that would be org.microbit.

Reply:
I agree.